core/processor/abci.go

```diff
@@ -2154,3 +2172,34 @@ func (app *App) DeliverEthereumKeyRotateSubmission(ctx context.Context, tx abci.
 		signatures.VerifyEthereumSignature,
 	)
 }
+
+func (app *App) CreateTeam(ctx context.Context, tx abci.Tx) error {
+	params := &commandspb.CreateTeam{}
+	if err := tx.Unmarshal(params); err != nil {
+		return fmt.Errorf("could not deserialize CreateTeam command: %w", err)
+	}
+
+	// The party that initiated this command has to retrieve the team ID through
+	// the API.
```

Comment:
Can we not use `addDeterministicID()` here to create the teamID from using the hash of the signature, like we do for proposals, orders etc. etc. Then any FE built against all this can find the teamID from their submission in the same way.

Reply:
Yes we can. We are just implementing this feature incrementally. I can add it now.